3rdparty/jvm/com/google/guava/BUILD

```diff
@@ -0,0 +1,8 @@
+jvm_artifact(
+    group="com.google.guava",
+    artifact="guava",
+    version="31.0.1-jre",
+    # Because `guava` provides a package which does not match its `group`, we explicitly list it here,
```

Comment:
The comment applies to acyclic as well but perhaps you're expecting thry read the whole repo at which point the comment becomes redundant?

Reply:
I think it makes sense to duplicate with `acylic` too. We shouldn't assume that people have read this already. Duplication is often a good thing in docs.